File: week-03/day-05/07.py
# create a function that takes a list and returns a new list with all the elements doubled

class Doubled:

    def reversed(self,elements):

        reverse = []
        for i in range( len(elements)):
            reverse.append(elements[i]*2)
        return reverse



numbers = [4,5,6,7]

magic = Doubled()


print(magic.reversed(numbers))

File: week-07/day-03/07.js
'use strict';

var numbers = [3,7,8,4];

// create a function that takes an array of numbers and returns a boolean
// it should return true if all the elements are prime, false otherwise

 function checkPrime(arr){

    var checkprimes = arr.map(function(e){
      var primes = true
      for(var i = 2; i <= Math.ceil(Math.pow(e,0.5)); i++){
         if(e % i == 0){
            primes = false;
            return primes;
         }
      }
      return primes
  })
  var ItHasPrime = checkprimes.every(function(e) {
    return e === true;
  });

  return ItHasPrime

 };

console.log(checkPrime(numbers));
